CONTRIBUTING.md

```diff
@@ -37,9 +37,9 @@ Feel free to check out the `#discussion`/`#development` channels on our [Slack](
 
 ## Developing
 
-_Node_: Check that Node is [installed](https://nodejs.org/en/download/) with version `^12.20 || >= 14.13`. You can check this with `node -v`.
+_Node_: Check that Node is [installed](https://nodejs.org/en/download/) with version `^20.10.0 || >=22.0.0`. You can check this with `node -v`.
 
-_Yarn_: Make sure that Yarn 1 is [installed](https://classic.yarnpkg.com/en/docs/install) with version >= `1.19.0`.
+_Yarn_: Make sure that Yarn is [installed](https://yarnpkg.com/getting-started/install) with version `>=4.0.0`.
```

Comment:
There seems to be no actual installation of yarn v4 here, unfortunately the yarn official website only leaves the corepack tutorial.
Let's change the wording to something like configure to support yarn v4?

Reply:
Well upon the first timing running `corepack enable yarn`, corepack will download the latest yarn binary and "install" it anyway. Contributors should refer to the official yarn docs for how to install yarn, hence the updated link. 

Therefore I think the current wording is fine, if we change it to something like "configure corepack to install yarn 4" but the yarn community one day decides to change their install recommendation, we will have to sync with them. For the time being, simply put "installed" seems sufficient.